resources/views/docs/1/publishing/building.md

```diff
@@ -49,15 +49,29 @@ Possible options are: `mac`, `win`, `linux`.
 
 **Cross-compilation is not supported on all platforms.**
 
+#### Cross-compilation on Linux
+
+Compiling Windows binaries is possible with [wine](https://www.winehq.org/).
+nsis requires 32-bit wine when building x64 applications.
```

Comment:
@RobertWesner please check here for a typo.

Reply:
Which typo do you mean? nsis not being capitalized?

```suggestion
NSIS requires 32-bit wine when building x64 applications.
```

Reply:
Looks much better.